.github/workflows/server.yml

```diff
@@ -76,23 +76,6 @@ jobs:
         run: |
           pip install -r examples/server/tests/requirements.txt
 
-      - name: Verify server deps
```

Comment:
Not super important right now, but it would be good to still have a workflow job to verify that `index.html` has been correctly generated and is up to date.

Reply:
Thanks for reminding, I added it back in https://github.com/ggerganov/llama.cpp/pull/10599/commits/b940cc8188b34aa428f140aeaa8294810c47d86d